### PR TITLE
Documentation for artipie system

### PR DESCRIPTION
**Pull request by g4s8**
#12 #28 #22 - added documentation for Artipie system

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#29](https://github.com/artipie/artipie/issues/29) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/d8e662de-80c8-41ab-909c-d106f0592794); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: README.md [hunk 1]
```diff
@@ -32,6 +32,82 @@ knows repo type (e.g. `maven` or `docker`) and storage settings for repo
...
+    # storage type
+    type: fs
+    path: /artipie/storage
+```
+
+Meta storage contains adapters configuration, where key is a repository name,
+and value is adapter config `yaml` file:
+```text
+config storage
+├── maven1
```
Comment (carlosmiranda) [inline README.md]: @g4s8 should this be `maven` instead?

File: README.md [hunk 2]
```diff
@@ -32,6 +32,82 @@ knows repo type (e.g. `maven` or `docker`) and storage settings for repo
...
+
+Artipie should be configured before startup.
+Main meta configuration `yaml` file should contains storage config,
+where adapter configuration files are located:
+```yaml
+meta:
+  # configuration storage
+  storage:
+    # storage type
+    type: fs
```
Comment (carlosmiranda) [inline README.md]: @g4s8 it's not clear what the names for the supported storage types should be, we should specify it somewhere.

File: README.md [hunk 3]
```diff
@@ -32,6 +32,82 @@ knows repo type (e.g. `maven` or `docker`) and storage settings for repo
...
+    username: admin
+    password: 123qwe
+```
+
+### Deployment
+
+Artipie web server can be started as standalone Java application, or
+started in a cluster with multiple instance behind load balancer.
+If Artipie was started in a cluster, all instances should receive
+single meta configuration for Artipie module. It's recommented to
```
Comment (carlosmiranda) [inline README.md]: @g4s8 Misspelling here, and slight reword suggestion.

> It's recommen_d_ed _that S3 storage be used_...

**Comment by g4s8** [comment]
@carlosmiranda thanks, please check new update

**Comment by 0crat** [comment]
Job audit: performer @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)[/z](https://www.0crat.com/u/carlosmiranda) didn't make CR comments

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
Order was finished: +15 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Pull request merged**